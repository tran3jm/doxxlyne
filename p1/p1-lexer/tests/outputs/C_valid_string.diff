--- outputs/C_valid_string.txt	2022-09-09 04:03:48.742026273 -0400
+++ expected/C_valid_string.txt	2022-09-09 03:44:24.954722331 -0400
@@ -1,3 +1,3 @@
 STRLIT   [line 001]  "kasdj293x"
-""
-"{+_}"
+STRLIT   [line 002]  ""
+STRLIT   [line 003]  "{+_}"
\ No newline at end of file
