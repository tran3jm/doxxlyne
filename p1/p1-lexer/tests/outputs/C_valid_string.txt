STRLIT   [line 001]  "kasdj293x"
""
"{+_}"
