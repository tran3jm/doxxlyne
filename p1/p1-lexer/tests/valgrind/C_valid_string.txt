==3456336== Memcheck, a memory error detector
==3456336== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3456336== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3456336== Command: ../decaf inputs/valid_string_c.decaf
==3456336== 
STRLIT   [line 001]  "kasdj293x"
""
"{+_}"
==3456336== 
==3456336== HEAP SUMMARY:
==3456336==     in use at exit: 0 bytes in 0 blocks
==3456336==   total heap usage: 338 allocs, 338 frees, 46,872 bytes allocated
==3456336== 
==3456336== All heap blocks were freed -- no leaks are possible
==3456336== 
==3456336== For lists of detected and suppressed errors, rerun with: -s
==3456336== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
