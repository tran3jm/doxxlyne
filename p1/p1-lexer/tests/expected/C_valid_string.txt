STRLIT   [line 001]  "kasdj293x"
STRLIT   [line 002]  ""
STRLIT   [line 003]  "{+_}"